src/routes/(app)/projects/+page.ts

```diff
@@ -0,0 +1,15 @@
+import { error } from '@sveltejs/kit';
+import type { PageLoad } from './$types';
+
+export const load: PageLoad = async ({ fetch }) => {
+  let res = await fetch(`/api/projects`);
+  if (!res.ok) error(res.status, 'No projects found');
```

Comment:
I think this way may be a bit misleading with the current message so I suggest this
```suggestion
  if (!res.ok) {
    if (res.status === 404) {
    error(404, 'No projects found');
  } else {
    error(res.status, 'Failed to load projects');
    }
  }

```